api/api.go

```diff
@@ -527,14 +541,22 @@ func (api *Server) Start() error {
 			log.L().Fatal("Node failed to serve.", zap.Error(err))
 		}
 	}()
+	if err := api.bc.AddSubscriber(api.blockListener); err != nil {
```

Comment:
Can you please double check blockchain is started before api and close after it

Reply:
Yes. Confirmed.